style.css

```diff
@@ -300,6 +300,72 @@ h2 {
   width: 30px;
 }
 
+.contact--form {
+  padding: 1rem;
+}
+
+.contact--section {
+  background-image: url(images/contact-shape.png), url(images/contact-bkground.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding-top: 103px;
+}
```

Comment:
- [ ] Ouch! this is close but it seems like the background image is not aligned properly as shown in the Figma design so I'd suggest you tweak it a little so it's aligned correctly.
- Your design:
  ![image](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/104968236/763d18aa-f0cd-4160-a7f5-3c17819414c5)
- Figma design:
  ![firefox_fgBx0GmBAh](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/104968236/7311da19-407e-48d0-be68-abe66694e323)
    - As you can see in your design some of the images are overlapping with the text and you also seem to be missing the icon beside the button. ✔